azure-functions/acbs-function/src/functions/activity-update-facility-covenant.js

```diff
@@ -41,6 +41,12 @@ const handler = async (payload) => {
 
     const { facilityIdentifier, acbsFacilityCovenantInput } = payload;
 
+    // As both fields on the payload are optional,
```

Comment:
Can we please remove this as this introduces another mechanism to validate a payload, though a good catch for a payload where both fields can be optional.

Reply:
updated